### ISSUE TITLE
FusedBatchNorm doesn't support is_training=True

### ISSUE THREAD

**Issue by kaixuanliu** [opencv/dldt#21]
Hi, I ran into a problem when converting a meta file to IR using MO, it shows "mo.utils.error.Error: FusedBatchNorm doesn't support is_training=True". How can I solve this problem besides offloading this operation?

**Comment by lazarevevgeny** [comment]
@kaixuanliu this error means that you provided _training_ graph to the Model Optimizer which is incorrect. 

Some background about this issue. TensorFlow produces different graph for training and for inference. The FusedBatchNorm is an example of an operation which behaves differently in training and inference mode (another example is Dropout operation). The Model Optimizer determined that the graph is training and exited with error because it cannot handle it.

The solution is to create the _inference_ graph. If you open Python script constructing the graph topology you will see that there should be a parameter like `is_training` which switch between inference and training graph generation. Change this value to `False` and create the pb file with the graph. 

In some models this switch is a placeholder in the topology. In this case you can convert the model using special command line parameter `--freeze_placeholder_with_value`. Refer to the example with FaceNet conversion https://software.intel.com/en-us/articles/OpenVINO-Using-TensorFlow

**Comment by kaixuanliu** [comment]
@lazarevevgeny Thanks! It is the same as what I guessed. Can close this issue now.
